File: src/gasing/division/division4.py
import time

a = 123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
b = 4

# GASING Division
start = time.time()
for _ in range(10000000):
        hasil = a * 25 // 100
end = time.time()
gasing_time = end - start
print("GASING Division 4:", gasing_time, "seconds")
print("GASING Result:", hasil)

# Computer Division
start = time.time()
for _ in range(10000000):
    hasil = a // b
end = time.time()
computer_time = end - start
print("General Division 4:", computer_time, "seconds")
print("Computer Result:", hasil)

print("\nGASING is", computer_time/gasing_time, "times faster")

File: src/gasing/division/division4.js
const a = 123456789;
const b = 4;

// GASING Division
let start = performance.now();
let hasil;
for (let i = 0; i < 10000000; i++) {
    hasil = Math.floor((a * 25) / 100);
}
let gasing_time = (performance.now() - start) / 1000;
console.log("GASING Division 4:", gasing_time, "seconds");

// Computer Division
start = performance.now();
for (let i = 0; i < 10000000; i++) {
    hasil = Math.floor(a / b);
}
let computer_time = (performance.now() - start) / 1000;
console.log("General Division 4:", computer_time, "seconds");

console.log("\nGASING is", computer_time/gasing_time, "times faster");